pkg/cli/subcommands/config/command.go

```diff
@@ -258,3 +228,33 @@ func getConfigValue(_ log.Component, config config.Component, cliParams *cliPara
 
 	return nil
 }
+
+func otelAgentCfg(_ log.Component, config config.Component, cliParams *cliParams) error {
+	if !config.GetBool("otelcollector.enabled") {
+		return errors.New("otel-agent is not present")
+	}
+	if !config.GetBool("otelcollector.converter.enabled") {
+		return errors.New("otel-agent converter must be enabled to get otel-agent's runtime configs")
+	}
+
+	err := util.SetAuthToken(config)
+	if err != nil {
+		return err
+	}
+
+	c, err := cliParams.GlobalParams.SettingsClient()
+	if err != nil {
+		return err
+	}
+
+	resp, err := util.DoGet(c.HTTPClient(), config.GetString("otelcollector.extension_url"), util.CloseConnection)
+	if err != nil {
+		return err
+	}
+	var extensionResp ddflareextensiontypes.Response
+	if err = json.Unmarshal(resp, &extensionResp); err != nil {
+		return err
+	}
+	fmt.Println(extensionResp.RuntimeConfig)
```

Comment:
Do we need to unmarshal the payload ? Could we simply print the response from the API ?

Reply:
I think it's better to unmarshal and only print the RuntimeConfig. The response contains a lot of other info like full env vars, build info, debug info etc. The raw response is also a bit hard to read